lldb/test/API/functionalities/mtc/swift/TestMTCSwift.py

```diff
@@ -45,6 +45,7 @@ def mtc_tests(self):
 
         self.expect(
             "thread info -s",
+            ordered=False,
```

Comment:
What makes the output nondeterministic? Is it not sufficient to reorder the substrings?

Reply:
This is the same.